### PR TITLE
What are you looking for

### PR DESCRIPTION
**Pull request by samhstn**
Adds "what are you looking for section" for https://github.com/LDMW/app/pull/100

### CODE REVIEW

File: home/models.py [hunk 1]
```diff
@@ -9,11 +10,13 @@
 
 class HomePage(Page):
     body = RichTextField(blank=True)
+    lookingfor = RichTextField(blank=True)
     alphatext = RichTextField(blank=True)
     footer = RichTextField(blank=True)
 
     content_panels = Page.content_panels + [
         FieldPanel('body', classname="full"),
+        FieldPanel('lookingfor', classname="full"),
```
Comment (Danwhy) [inline home/models.py]: Some help text here (and maybe some for 'alphatext' too) might be a good idea, just to make it clear what this part of the homepage is

Comment (samhstn) [reply home/models.py]: I'll add that in

**Review by nelsonic — APPROVED** [review]
@shouston3 LGTM. 👍

**Review by SimonLab — APPROVED** [review]
Looks good. @shouston3 do you want to add the help text on this PR (I don't think they have been added yet? see @Danwhy comment) or on another one, ie is this PR ready to merge or should I wait?

**Comment by samhstn** [comment]
@SimonLab Added

**Pull request merged**